docs/server/source/server-reference/configuration.md

```diff
@@ -533,6 +535,22 @@ logging of the `core.py` module to be more verbose, you would set the
 **Defaults to**: `"{}"`
 
 
+### log.port
```

Comment:
I know this isn't something you did but I just noticed that three lines above this, it says log.granular_levels defaults to `"{}"` but that's not quite right. It's not a string. It defaults to `{}`. We may as well change it in this PR, since it's such a minor cosmetic fix.

Reply:
sure